backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -222,13 +223,16 @@ def _generate_privilege_record(
         jurisdiction_postal_abbreviation: str,
         license_expiration_date: date,
         compact_transaction_id: str,
+        original_issuance_date: date | None = None,
     ):
+        today = datetime.now(tz=UTC).date()
```

Comment:
UI: How do I get back to the dashboard when on the Account page (for providers)?

<img width="1510" alt="image" src="https://github.com/user-attachments/assets/26cb3a19-afc3-40a2-bbc8-f34c2d5b2e38">

@jsandoval81